website/snippets/_sl-test-and-query-metrics.md

```diff
@@ -4,44 +4,41 @@ This section explains how you can test and run MetricFlow commands with dbt Clou
-   - MetricFlow builds a semantic graph and generates a `semantic_manifest.json` file in dbt Cloud, which is stored in the `/target` directory. If using the Jaffle shop example, run `dbt seed && dbt run` to ensure the required data is in your data platform before proceeding.
+Refer to the following steps to get started:
 
-4. Run `dbt sl --help` to confirm you have MetricFlow installed and that you can view the available commands.
-5. Run `dbt sl query --metrics <metric_name> --group-by <dimension_name>` to query the metrics and dimensions. For example, `dbt sl query --metrics order_total --group-by metric_time`
-6. Verify that the metric values are what you expect. To further understand how the metric is being generated, you can view the generated SQL if you type `--compile` in the command line.
-7. Commit and merge the code changes that contain the metric definitions.
+1. **Installation** &mdash; Make sure you've installed the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation). Then, navigate to your dbt project directory.
+2. **Initial command** &mdash; Run a dbt command, such as `dbt parse`, `dbt run`, `dbt compile`, or `dbt build`. If you don't, you'll receive an error message that begins with: "ensure that you've ran an artifacts....".
+3. **Semantic graph and manifest** &mdash; MetricFlow builds a semantic graph and generates a `semantic_manifest.json` file in dbt Cloud, which is stored in the `/target` directory. If using the Jaffle shop example, run `dbt seed && dbt run` to ensure the required data is in your data platform before proceeding.
 
-To streamline your metric querying process, you can connect to the [dbt Semantic Layer APIs](/docs/dbt-cloud-apis/sl-api-overview) to access your metrics programmatically. For SQL syntax, refer to [Querying the API for metric metadata](/docs/dbt-cloud-apis/sl-jdbc#querying-the-api-for-metric-metadata) to query metrics using the API.
+:::info Run dbt parse when you make changes to metrics
+Anytime you make changes to metrics, you need to run `dbt parse` at the minimum. This ensures the `semantic_manifest.json` file is updated and you can have your changes reflected when querying metrics.
+:::
 
+4. **MetricFlow confirmation** &mdash; Run `dbt sl --help` to confirm you have MetricFlow installed and that you can view the available commands.
+5. **Query metrics** &mdash; Run `dbt sl query --metrics <metric_name> --group-by <dimension_name>` to query the metrics and dimensions. For example, `dbt sl query --metrics order_total --group-by metric_time`.
+6. **Metrics verification** &mdash; Verify that the metric values are what you expect. To further understand how the metric is being generated, you can view the generated SQL if you type `--compile` in the command line.
+7. **Commit changes** &mdash; Commit and merge the code changes that contain the metric definitions.
```

Comment:
sugg: update constructions for these steps for consistency

Reply:
updated teh steps and folded in - can yourereview @nghi-ly ! tysm!